packages/@spectrum-icons/ui/scripts/generateIcons.js

```diff
@@ -34,7 +34,7 @@ function template(iconName) {
     let jsx = compileSVG(path.join(expressDir, expressName + '.svg'), 'ExpressIcon');
 
     return (
-`import {${iconName} as IconComponent} from '@adobe/react-spectrum-ui/dist/${iconName}';
+`import {${iconName} as IconComponent} from '@adobe/react-spectrum-ui/dist/${iconName}.js';
```

Comment:
@adobe/react-spectrum-ui only provides CJS, so we include the file extension to force any loaders to recognize that as opposed to everywhere else where we let the loader choose which one it wants to use